index.md

```diff
@@ -0,0 +1,31 @@
+This file lists the available Mattermost plugins.
+
+To keep it Bot-parseable, only plugin listings can include `[]()` formatted URL references.
+
+#### Examples
+- [mattermost-plugin-sample](https://github.com/mattermost/mattermost-plugin-sample.git)
+- [mattermost-plugin-demo](https://github.com/mattermost/mattermost-plugin-demo.git)
+
+#### Production
```

Comment:
I propose to rename this to `Official`. `Production` implies that the plugins under `Unofficial` are't ready for production.

1/5 on the name `Official`
4/5 that `Production` is a bad name.

Reply:
Does `Official` imply `Certified`?

Reply:
Separate PR?